### PR TITLE
GBOOK-4: Add admin page.

### PR DESCRIPTION
**Pull request by bettyMelania**

### CODE REVIEW

File: Controllers/AdminController.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  {
+    parent::__construct();
+    $this->title = "Admin page";
+  }
+
+  /**
+   * Callback for /admin route.
+   */
+  public function adminPageAction() {
+    $this->content = $this->render('/views/layouts/admin_content.php');
```
Comment (laurentiu1981) [inline Controllers/AdminController.php]: Please rename layout file to admin_content.tpl.php and move it under /views/admin folder.
We'll use /views/layouts folder for full pages templates.

File: Controllers/AdminController.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    $sidebar = $this->render('/views/layouts/admin_sidebar.php');
```
Comment (laurentiu1981) [inline Controllers/AdminController.php]: Considering that this template contains a search form, it should be renamed to something that suggests that and moved under /views/forms. For example: admin_search_form.tpl.php
Ideally, any template file should use .tpl.php extension.

File: views/layouts/admin_sidebar.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  <div class="form-group">
+    <label for="author">Authors:</label>
+    <select class="form-control" id="author" >
+      <option>Select Author</option>
+    </select>
+  </div>
+
+  <div class="form-group">
+    <label for="price">Price</label>
```
Comment (laurentiu1981) [inline views/layouts/admin_sidebar.php]: "for" attribute needs to target a particular id. If you have for="price", it is expected to have an element with that id.
The effect is that if you click on label text, the element with id provided in "for" attribute will be focused. In this case it has no target.

File: views/layouts/admin_sidebar.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    <input type="number" class="form-control" id="price_from" placeholder="from" min="0">
+    <input type="number" class="form-control" id="price_to" placeholder="to" min="0">
```
Comment (laurentiu1981) [inline views/layouts/admin_sidebar.php]: "id" and "class" attributes should use dashes instead of underscore. Underscore is not supported by all browsers correctly.
In this case: price_from / price_to need to be updated to price-from and price-to
Every input field should have a name. It's the attribute that is passed in $_POST on submit.

File: Controllers/AdminController.php [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Controllers;
+
+class AdminController  extends BasicController
```
Comment (laurentiu1981) [inline Controllers/AdminController.php]: Apply coding standards: two spaces before extends.
Please use Ctrl + Alt + L before push. It will automatically remove extra white spaces.

File: views/layouts/admin_content.php [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+  <thead>
+  <tr>
+    <th>Icon</th>
+    <th>Title</th>
+    <th>Deeplink</th>
+    <th>Author</th>
+    <th>Options</th>
+  </tr>
+  </thead>
+
```
Comment (laurentiu1981) [inline views/layouts/admin_content.php]: Please remove unneeded empty lines.

File: Controllers/AdminController.php [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    $this->title = "Admin page";
+  }
+
+  /**
+   * Callback for /admin route.
+   */
+  public function adminPageAction() {
+    $this->content = $this->render('/views/layouts/admin_content.php');
+    $sidebar = $this->render('/views/layouts/admin_sidebar.php');
+    $this->renderLayout('/views/layouts/sidebar_page.tpl.php',array('sidebar'=>$sidebar));
```
Comment (laurentiu1981) [inline Controllers/AdminController.php]: Apply coding standards. empty spaces missing in multiple places in this line.

File: views/layouts/sidebar_page.tpl.php [hunk 8]
```diff
@@ -7,6 +7,7 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <link href="/css/style.css" rel="stylesheet" />
```
Comment (laurentiu1981) [inline views/layouts/sidebar_page.tpl.php]: Hmm, this line seems extra and not related to your work, it seems.

File: views/layouts/admin_sidebar.php [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+  </div>
+
+  <button type="submit" class="btn btn-primary btn-md center-block "><span class="glyphicon glyphicon-search"></span>Search</button>
+
```
Comment (laurentiu1981) [inline views/layouts/admin_sidebar.php]: Please remove unneeded empty lines.

File: views/layouts/sidebar_page.tpl.php (hunk 8 above)
Comment (bettyMelania) [reply views/layouts/sidebar_page.tpl.php]: yes, but the menu isnțt horizontal on admin page, if I don't include the css file

Comment (laurentiu1981) [reply views/layouts/sidebar_page.tpl.php]: Ok to stay in this case.
